spec/models/food_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  subject do
+    @user = User.create(name: 'Faranosh', email: 'faranosh@mail.com', password: 'password')
+    Food.create(name: 'Corn', measurement_unit: 'grams', price: 2, user: @user)
+  end
+
+  before(:each) { subject.save }
+
+  it 'should have name corn' do
+    expect(subject.name).to eql('Corn')
+  end
+
+  it 'should have Measurement unit grams' do
+    expect(subject.measurement_unit).to eql('grams')
+  end
+
+  it 'Should always have Measurement unit present' do
+    subject.measurement_unit = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'should have price of 2' do
+    expect(subject.price).to eql(2)
```

Comment:
- [ ] Great job working on the testing for the models but I think you should also add unit tests for the views and controllers as well since you should test your app if it is working properly through this testing so kindly add them and make sure they are running well ✅💯⭐🔥 

![image](https://github.com/Tafloninno/recipe-app/assets/58553711/c624832c-6ddf-4fd8-a964-b75ee7c0d72f)